src/main/py/trade/offer_response.py

```diff
@@ -14,31 +14,34 @@ class OfferResponse:
     def __init__(
         self,
         trade_id: str,
-        artwork_ledger_id: str,
+        artwork_ledger_key: str,
         originator_public_key: str = "",
     ):
         """
         Initializes an instance of the OfferResponse class.
         - trade_id (str): The trade id to respond to.
         """
         self.trade_id = trade_id
-        self.offer_ledger_id = artwork_ledger_id
+        self.artwork_ledger_key = artwork_ledger_key
```

Comment:
To me, `artwork_id` is more clear for this since it's not a `key` in the same way the `originator` has a key. It's more like a GUID (it's created by generating a random sha1 hash).

I think `OfferAnnouncement` should be changed to the same to achieve the consistency you were going for.